README.md

```diff
@@ -4,8 +4,8 @@
 
 | Type | Link       | 
 |------|------------|
-| prod | <https://proj-gauchoride.dokku-00.cs.ucsb.edu/> | 
-| qa | <https://proj-gauchoride-qa.dokku-00.cs.ucsb.edu/>  | 
```

Comment:
This shoudl be in it's own 5 point separate PR, not mixed in with this one.  Please remove it from this one.

PRs should deal with one concern.